src/config/api.js

```diff
@@ -2,7 +2,7 @@ import { create } from 'apisauce';
 
 const baseURL = process.env.REACT_APP_BASE_URL;
 
-if (baseURL === process.env.REACT_APP_BASE_URL) {
+if (baseURL === 'http://wolox.com' || !baseURL) {
```

Comment:
Nice one! You'll only display the `console.warn` when there's an actual problem! 💯